1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -2,6 +2,7 @@ export const fizzbuzz = (value: number): string => {
   if (isNaN(value) || value < 1 || value > 100) return "";
 
   if (value % 3 === 0) return "Fizz";
+  if (value % 5 === 0) return "Buzz";
```

Comment:
same here as the previous comment

Reply:
It was a struggle restraining myself to moving in smaller steps especially since I had the solution in my head 😄. 

I get the general idea of what you mean, but I initially struggled a bit to narrow down the specific point in my commits you were giving feedback on, it would have been more helpful if you referenced the commit hash in your comment. 

Thanks for the feedback! I'll be taking it into my next iteration of this or the next assignment.